### ISSUE TITLE
Unable to connect to graphqlhub.com

### ISSUE THREAD

**Issue by mattshadowwing** [leebenson/reactql#189]
Hello, I was trying out ReactQL today and after installing all dependencies and loaded the browser for the first time, i was met with this error in the console:

`FetchError: request to https://graphqlhub.com/graphql failed, reason: connect ECONNREFUSED `

**Comment by aoede3** [comment]
fyi https://github.com/clayallsopp/graphqlhub/issues/53#issuecomment-694440831

**Comment by leebenson** [comment]
I'm not currently maintaining this starter kit. The last update was over a year ago, so third-party resources like this are likely to be redundant, and deps outdated.

Would recommend trying something else, or even better, rolling your own per your requirements. I don't think starter kits are very useful in production.

**Comment by cirosantilli** [comment]
It would be great if there were simple instructions to run/populate the graphql server locally. Shame to see such promising work hidden behind a possibly relatively small entry barrier.

**Comment by leebenson** [comment]
@cirosantilli, thanks for your interest! My last commit to this project was almost 2 years ago. Even getting a local GraphQL instance running would likely be out of date due to missing or redundant libs. I'd recommend looking at other maintained libraries and starter kits rather than investing the effort into this right now. Thanks for checking it out, anyway!

**Comment by cirosantilli** [comment]
I understand Lee. When I ramp down my projects, I generally try to leave a Docker setup for future readers. But of course, you have absolutely no obligation of doing anything.

Any suggestions of other projects doing similar things BTW?

**Comment by leebenson** [comment]
There are so many interesting projects out there since creating this starter kit, that it really depends what you're trying to achieve.

For a general React framework, I'd probably go with NextJS. Hard to argue with the funding and commercial support being ploughed into it. It's also a first target for React Server, which makes it more interesting to me. There's a far higher likelihood the framework will be supported and upgraded vs. a solo-author starter kit like ReactQL.

For bolting on GraphQL, I'd choose either Url or Apollo on the client side. I tend to write GraphQL servers in Rust or .NET these days, but Apollo Server is an obvious choice for JS.

I use [GraphQL Code Generator](https://www.graphql-code-generator.com/) for generating types.

Beyond that, there's a bunch of interesting new libs. If you're doing a lot of DB access and want to build a GraphQL server on top of it, [Prisma](https://www.prisma.io/) looks like a good JS choice with some commercial backing. I haven't used it in a project, so no personal experience with it.

In short-- I don't think starter kits are required, or make a whole lot of sense at a certain point. I built this one because I found myself reaching for the same tools over multiple projects. My preferred stack has changed since that time, especially for back-end projects, and I found myself choosing new tools that made this starter kit redundant.

For new front-end projects, I'd probably just go with NextJS, and layer what I need on top. For back-end, I'd choose something other than JS.
